compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/ConfigMethodGen.java

```diff
@@ -104,35 +108,44 @@ public void generateConfigMapper(List<PackageID> imprtMods, BIRNode.BIRPackage p
         JvmCodeGenUtil.visitMaxStackForMethod(mv, JVM_INIT_METHOD, innerClassName);
         mv.visitEnd();
 
-        generateConfigInit(cw, moduleInitClass, imprtMods, pkg.packageID);
+        generateConfigInit(cw, imprtMods, pkg.packageID, innerClassName);
 
         populateConfigDataMethod(cw, moduleInitClass, pkg, new JvmTypeGen(jvmConstantsGen, pkg.packageID,
                 typeHashVisitor, symbolTable));
         cw.visitEnd();
         jarEntries.put(innerClassName + CLASS_FILE_SUFFIX, cw.toByteArray());
     }
 
-    private void generateConfigInit(ClassWriter cw, String moduleInitClass, List<PackageID> imprtMods,
-                                    PackageID packageID) {
+    private void generateStaticFields(ClassWriter cw, String innerClassName) {
+        MethodVisitor mv = cw.visitMethod(ACC_STATIC, JVM_STATIC_INIT_METHOD, VOID_METHOD_DESC, null, null);
```

Comment:
Shall we move `"Z"` to a constant in `JvmSignatures` and use it here, since we have multiple use cases? Let's replace in all the other places too.

Reply:
Shall we use `GET_JBOOLEAN_TYPE` as the constant name for the `"Z"` type descriptor? 

Please let me know if there is a better name for it.

Reply:
Used `GET_JBOOLEAN_TYPE` as the constant name in 8da28c04c1397a39613ef773b0f56c5989ce3016. Please update here if the name should be changed.

Reply:
`GET_JBOOLEAN_TYPE` seems ok to me.